eq-author/cypress/support/commands.js

```diff
@@ -10,8 +10,25 @@ Cypress.Commands.add("login", options => {
     picture: "",
   };
 
-  const accessToken = createAccessToken(tokenPayload);
-  window.localStorage.setItem("accessToken", accessToken);
+  cy.window()
+    .its("__auth__")
+    .then(auth => {
+      return new Promise(res => {
+        // when firebase has triggered default logout
+        auth.onAuthStateChanged(() => {
+          setImmediate(() => {
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/Window/setImmediate
I found out this is a non-standard API and requires a so we should replace it with `setTimeout(fn, 0)`